resources/session03/wsgi/wsgi-calc/calculator.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import re
+
+
+def addition(num_str):
+    if num_str.endswith('/'):
+        num_str = num_str.rstrip('/')
```

Comment:
No need to check endswith. If the string does not end in '/', rstrip does nothing.